src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/PrisonApiClientTest.kt

```diff
@@ -482,8 +482,8 @@ class PrisonApiClientTest {
     val offenderNumber = "ABC123A"
 
     mockServer.stubErrorTemporaryAbsencesSuccess(offenderNumber, 400)
-    try {
-      val response = prisonApiClient.confirmTemporaryAbsencesArrival(
+    runCatching {
```

Comment:
nice! did we have a similar pattern in a PR from the last few days where we were expecting an exception? if so, should we reuse the same mechanism?